packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -7,11 +7,13 @@ import 'dart:async';
 import 'package:args/command_runner.dart';
 
 import '../base/common.dart';
+import '../base/context.dart';
 import '../base/file_system.dart';
 import '../base/terminal.dart';
 import '../base/time.dart';
 import '../base/utils.dart';
 import '../build_info.dart';
+import '../build_runner/web_fs.dart';
```

Comment:
You can't expose anything from a build_runner folder through the tool normally, it needs to go through an indirection of some kind. The issue is that the build_runner deps don't exist in google3 or fuchsia-source, so there is a risk of the tool not building

Reply:
The only thing being imported is a simple `typedef`. I need it so I can inject one into context (see https://github.com/flutter/flutter/blob/3ee2f16793fab9a96768ab5f2c69032fbfb387ee/packages/flutter_tools/lib/src/commands/run.dart#L383-L385).

Is there somewhere suitable I can move that typedef that would allow me to remove this? (I guess it could pretty much be anywhere, I just couldn't find anything similar to be consistent with.

Reply:
It doesn't matter what you import, because it will force all of the transitive imports to be resolved which will fail once it hits a build_runner/daemon import.

I would place it somewhere in base (net.dart?)